code/modules/antagonists/traitor/medrp/medrp_traitor.dm

```diff
@@ -0,0 +1,117 @@
+			if(hijacking)
+				return "Gone Postal"
+			else
+				return pickweight(list("Internal Affairs Agent" = 3, "Corporate Climber" = 1, "Legal Trouble" = 1))
+
+//moved to the bottom of the file because of it's size
+/datum/antagonist/traitor/roleplay/proc/fluff_strings()
+	. = list()
+	switch(role_title)
+		if("Tiger Cooperative")
+			. += "<span class='userdanger'>You are the Tiger Cooperative Fanatic.</span>"
+			. += "<span class='big bold'>Only the enlightened Tiger brethren can be trusted; all others must be expelled from this mortal realm!</span>"
+			. += "<span class='notice'>Remember the teachings of Hy-lurgixon; kill first, ask questions later!</span>"
+			. += "<span class='notice'>You have been provided with a standard uplink to prove yourself to the changeling hive. If you accomplish your tasks, you will be assimilated.</span>"
+		if("Waffle Corporation Terrorist")
+			. += "<span class='userdanger'>You are the Waffle Corporation Terrorist.</span>"
+			. += "<span class='big bold'>Most other syndicate operatives are not to be trusted, except for members of the Gorlex Marauders. Do not trust fellow members of the Waffle.co (but try not to rat them out), as they might have been assigned opposing objectives.</span>"
+			. += "<span class='notice'>It has been a relatively quiet month, many of our black market weapons have gone unused. Let's give Nanotrasen a live test and hijack one of their shuttles.</span>"
+			. += "<span class='notice'>You have been provided with a standard uplink to accomplish your task.</span>"
+		if("Animal Rights Consortium")
+			. += "<span class='userdanger'>You are the ARC Terrorist.</span>"
+			. += "<span class='big bold'>You may cooperate with other syndicate operatives if they support our cause. Maybe you can convince the Bee Liberation Front operatives to cooperate for once?</span>"
+			. += "<span class='notice'>The creatures of this world must be freed from the iron grasp of Nanotrasen, and you are their only hope!</span>"
+			. += "<span class='notice'>The Syndicate have graciously given one of their uplinks for your task.</span>"
+		if("Bee Liberation Front")
+			. += "<span class='userdanger'>You are the Bee Liberation Front Operative.</span>"
+			. += "<span class='big bold'>You may cooperate with other syndicate operatives if they support our cause. Maybe you can recruit an Animal Rights Consort to be useful for once?</span>"
+			. += "<span class='notice'>We must prove ourselves to the Syndicate or we will not be able to join. Animal Rights Consort will roll us!</span>"
+			. += "<span class='notice'>The Syndicate have graciously given one of their uplinks to see if we are worthy.</span>"
+		if("Cybersun Industries")
+			. += "<span class='userdanger'>You are from Cybersun Industries.</span>"
+			. += "<span class='big bold'>All other syndicate operatives are not to be trusted. Fellow Cybersun operatives are to be trusted. Members of the MI13 organization can be trusted.</span>"
```

Comment:
```suggestion
			. += "<span class='big bold'>Fellow Cybersun operatives are to be trusted. Members of the MI13 organization can be trusted. All other syndicate operatives are not to be trusted. </span>"
```
I know this is how its written on the wiki but its bad grammar

Reply:
Sure